lib/govuk_index/publishing_event_message_handler.rb

```diff
@@ -15,57 +13,56 @@ class NotIdentifiable < StandardError; end
 
   class MissingExternalUrl < StandardError; end
```

Comment:
These classes should also be within the PublishingEventMessageHandler namespace (or if they can't be there should probably be in a lib/govuk_index.rb file)